### ISSUE TITLE
Could not resolve reference: Xamarin.Android.Support.Fragment, Version 1.0.0.0

### ISSUE THREAD

**Issue by wtheronjones** [rotorgames/Rg.Plugins.Popup#233]
I installed the latest package from nugget (1.1.4.129-beta), and it worked great in iOS, but in Android, I am getting errors about resolving references.  Sadly, I can't use this library if I can't use it on Android.

I built w/ MSBuild output set to Diagnostic (Tools->Options->Projects & Solutions->Build & Run) and have some snippets:
6>  Dependency "Xamarin.Android.Support.Fragment, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null". (TaskId:728)
6>      Could not resolve this reference. Could not locate the assembly "Xamarin.Android.Support.Fragment, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null". Check to make sure the assembly exists on disk. If this reference is required by your code, you may get compilation errors. (TaskId:728)

  Required by "Xamarin.Forms.Xaml, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL". (TaskId:728)
6>      Required by "Xamarin.Forms.Core, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL". (TaskId:728)
6>      Required by "Xamarin.Forms.Platform, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL". (TaskId:728)
6>      Required by "Rg.Plugins.Popup.Droid, Version=1.1.4.129, Culture=neutral, processorArchitecture=MSIL". (TaskId:728)
6>      Required by "Rg.Plugins.Popup, Version=1.1.4.129, Culture=neutral, processorArchitecture=MSIL". (TaskId:728)
6>      Required by "Xamarin.Forms.Platform.Android, Version=2.0.0.0, Culture=neutral, processorArchitecture=MSIL". (TaskId:728)

Xamarin.Android.Common.targets(1500,2): error : Exception while loading assemblies: System.IO.FileNotFoundException: Could not load assembly 'Xamarin.Android.Support.Fragment, Version=1.0.0.0, Culture=neutral, PublicKeyToken='. Perhaps it doesn't exist in the Mono for Android profile?

Note that we use Xamarin.GooglePlayServices.Base/Basement/SafetyNet, one of which references Xamarin.Android.Support.v4.23.1.1.1, and installing this package updates it to v4.23.3.0

**Comment by rotorgames** [comment]
@wtheronjones Try to load the Demo project from this solution and build it.

**Comment by wtheronjones** [comment]
@rotorgames Yes, I tried that, and I can't build Demo.Droid. I created another issue for that one.

**Comment by rotorgames** [comment]
@wtheronjones When I get `FileNotFoundException` I just build project one after another. In first I build PCL/.NET Standard project (Rg.Plugins.Popup), after it I build Android project of this library (Rg.Plugins.Popup.Droid), after it I build Demo project and after it I build Demo.Droid project. I don't know what happens but it works. It is not problem of this plugin, it is xamarin's problem.

P.S. Don't forget to do all items from [Troubleshooting](https://github.com/rotorgames/Rg.Plugins.Popup#doesnt-workbuild-or-etc) before it.

**Comment by wtheronjones** [comment]
@rotorgames I resolved this error by upgrading the Android Compatibility references.  When Xamarin.Forms is installed, the compatibility libraries are 23.3.0, and I get this error.

I upgraded those libraries to 25.4.0.2.  note that nuget put right package in place, but in the csproj file it references the Xamarin.Android.Support.Annontations by the wrong version # and you get a build error. I manually removed that from the EnsureNuGetPackageBuildImports target section in the csproj.
